odysseus/README.md

```diff
@@ -41,10 +46,14 @@ In any terminal that is in the directory:
 
 ## Building the Project
 1. Run `make <config>` using the config you want to build (see above), note that any `menuconfig` changes are overwritten with this command.
-2. Run ```make``` (Note: this can take a few hours on first build, subsequent builds take less time)
+2. Run ```make -j$(($(nproc)+1)) --output-sync=target``` (Note: this can take a few hours on first build, subsequent builds take less time).  Lower the -jN number to use less cores of your CPU to make your system usable during the build, at the expense of time.
```

Comment:
so `make -j$(($(nproc)+1))` did not work for me. I think you should just change it to `make -j$(nproc)` for system and shell versatility.

Reply:
Alright.  Some docs says nproc+1 number of cores but it's really anecdotal.